File: mvp-safco-ui/src/app/services/models/equipo-response.ts
/* tslint:disable */
/* eslint-disable */
export interface EquipoResponse {
  capacidadDiscoD?: string;
  caracDiscoD?: string;
  caracRAM?: string;
  codInterno?: string;
  descripcionProce?: string;
  empresaCompra?: string;
  estadoEquipo?: 'RENOVADA' | 'DEVUELTA' | 'OPERATIVA' | 'ROBADA';
  fechaCompra?: string;
  idEquipo?: number;
  licenciaAntivirus?: string;
  licenciaOffice?: string;
  linea?: string;
  macWifi?: string;
  marca?: string;
  memoriaRam?: string;
  modelo?: string;
  modeloCargador?: string;
  procesador?: string;
  serie?: string;
  serieCargador?: string;
  tipoEquipo?: 'ALQUILADA' | 'SAFCO';
}

File: mvp-safco/src/main/java/com/safco/mvp_safco/models/responses/EquipoResponse.java
package com.safco.mvp_safco.models.responses;

import com.safco.mvp_safco.models.enums.EstadoEquipo;
import com.safco.mvp_safco.models.enums.TipoEquipo;
import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDate;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class EquipoResponse {

    private Long idEquipo;

    private String codInterno;
    private String empresaCompra;
    private String linea;
    private String marca;
    private String modelo;
    private String serie;
    private String procesador;
    private String descripcionProce;
    private String memoriaRam;
    private String caracRAM;
    private String capacidadDiscoD;
    private String caracDiscoD;
    private String modeloCargador;
    private String serieCargador;
    private LocalDate fechaCompra;
    private String macWifi;
    private String licenciaOffice;
    private String licenciaAntivirus;

    private EstadoEquipo estadoEquipo;
    private TipoEquipo tipoEquipo;
}
